blockchain.go

```diff
@@ -1042,11 +1039,13 @@ func (b *Blockchain) GetAccountStorage(address sdk.Address) (*AccountStorage, er
 		nil,
 	)
 
+	r := b.vmCtx.Borrow(env)
+	defer b.vmCtx.Return(r)
```

Comment:
@janezpodhostnik Is this safe? I'm not keeping up with FVM changes, but from what I see, it looks like we keep on using the runtime after the function returned (through the storage that is returned from this function)

Reply:
Good point. Let me take a look and see if we can make this safer. One option is to simply never return it